packages/client/src/forms/configuration/administrative/addresses.ts

```diff
@@ -383,6 +383,7 @@ export const getXAddressSameAsY = (
     label,
     required: true,
     initialValue: true,
+    previewGroup: AddressSubsections.PRIMARY_ADDRESS_SUBSECTION,
```

Comment:
@Zangetsu101 doesnt this need to be dynamic?

Reply:
Well we only have this field for primary addresses. So I don't think we need it to be dynamic for now